### PR TITLE
Add home page link to nav bar

### PR DESCRIPTION
**Pull request by Brookke**
Not sure if the var name homeurl is descriptive enough?

Closes: #583

### CODE REVIEW

File: src/Templates/parts/nav.twig [hunk 1]
```diff
@@ -10,6 +10,7 @@
       </button>
       {% endif %}
       <a class="navbar-brand" href="{{baseurl}}">MyRadio</a>
+      <a class="navbar-brand" href="{{homeurl}}">ury.org.uk</a>
```
Comment (LordAro) [inline src/Templates/parts/nav.twig]: Not really much point in making it configurable if you just hardcode the text anyway?

File: src/Classes/Config.php [hunk 2]
```diff
@@ -80,6 +80,13 @@
     public static $schedule_url = '//ury.org.uk/schedule';
 
     /**
+     * The base URL of the radio home pages.
+     *
+     * @var string
+     */
+    public static $home_url = '//ury.org.uk/';
```
Comment (AJDurant) [inline src/Classes/Config.php]: Config variables like this should really be declared here and set in the local config.

File: src/Templates/parts/nav.twig (hunk 1 above)
Comment (AJDurant) [inline src/Templates/parts/nav.twig]: It shouldn't have 'navbar-brand' read the Bootstrap docs.
And it should probably go into the hamburger menu on small screens.

Comment (Brookke) [inline src/Templates/parts/nav.twig]: @LordAro I did mention that, I can also create a var for the name.

@AJDurant I will sort that now

Comment (AJDurant) [inline src/Templates/parts/nav.twig]: @Brookke you've already got `$short_name` and `$long_name` in the config you could use (probably shortname). Maybe '`short_name` Website' or something.

File: src/Classes/Config.php (hunk 2 above)
Comment (Brookke) [inline src/Classes/Config.php]: @AJDurant How do I go about setting that in the local config? Ie how is that generated?

**Comment by Brookke** [comment]
I think I have done everything asked except the local config thing, however it doesn't seem like the short name or long name are set in the local config either?

**Comment by AJDurant** [comment]
Yeah, a load of stuff should be moved there. It's not a major issue. I just checked it out on -dev looks good 👍

**Comment by Brookke** [comment]
Thanks @AJDurant 😄

**Pull request merged**